cmd/webhook/app/webhook.go

```diff
@@ -133,6 +135,12 @@ func handle() {
 				&admissionregistrationv1.MutatingWebhookConfiguration{}: {
 					Field: fields.SelectorFromSet(fields.Set{"metadata.name": common.WebhookName}),
 				},
+				// restrict the number of configmap to be cached.
+				&v1.ConfigMap{}: {
+					Label: labels.SelectorFromSet(labels.Set{
```

Comment:
I suggest adding a custom label (e.g., fluid.io/configmap-type: worker-state) to ConfigMaps that track Worker Pod states. The purpose is making controller cache only tracking very few configmaps.